configure_wifi.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/sh
 
-cmake -B build -S . -DCMAKE_EXPORT_COMPILE_COMMANDS=Y -GNinja -DPICO_BOARD=pico_w -DPICO_SDK_PATH=/home/holz/etc/git/pico-sdk/ -DPICO_EXTRAS_PATH=/home/holz/etc/git/pico-extras/ -DWIFI_SSID=$(awk -F '=' '$1=="ssid" {print $2}' wlan.ini) -DWIFI_PASSWORD=$(awk -F '=' '$1=="pass" {print $2}' wlan.ini)
+cmake -B build -S . -DCMAKE_EXPORT_COMPILE_COMMANDS=Y -GNinja -DPICO_BOARD=pico_w
+-DPICO_SDK_PATH=${PICO_SDK_PATH} -DPICO_EXTRAS_PATH=${PICO_EXTRAS_PATH}
+-DWIFI_SSID=$(awk -F '=' '$1=="ssid" {print $2}' wlan.ini)
+-DWIFI_PASSWORD=$(awk -F '=' '$1=="pass" {print $2}' wlan.ini)
```

Comment:
I think we are missing backslashes, otherwise these will be treated as separate commands.